docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -0,0 +1,134 @@
+---
+id: dt-kit-interaction-patterns
+title: Interaction Patterns
+description: 'Digital Twin KIT - Interaction Patterns'
+sidebar_position: 2
+---
+
+## Interaction Patterns for Distributed Digital Twins
+
+The Catena-X standard CX-0002 (Digital Twins in Catena-X) defines a subset of the
+
+### 1. Fetching a supplier's Twin
+
+| Scenario                                                                    | Participants                                           | Assumptions                                                                 | Links to Use-Cases                                                                               |
+|-----------------------------------------------------------------------------|--------------------------------------------------------|-----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| The supplier has data required by the OEM. For instance a Bill of Material. | - Data Provider (Supplier) <br/> - Data Consumer (OEM) | 1. Consumer knows Supplier's BPN <br/> 2. Consumer knows an id of the asset | - Industry Core (coming soon) <br/> - [Product Carbon Footprint Kit](../../PCF%20Exchange%20Kit) |
+
+The most common integration pattern with Digital Twins aims to offer asset-related data in expectable ways along the
+supply chain. This scenario assumes that the supplier offers data about an asset that was sold to an OEM further up the
+chain. Examples for this may include Bills of Material, carbon footprints, technical specifications or regulatory 
+documents. An OEM - or really any interested and privileged third party - can assume that a part's supplier offers a Digital
+Twin if the data-exchange is based on the Digital Twin Kit (and by proxy CX-0002). The data provider deploys at least:
+- Digital Twin Registry (DTR)
+  - registered in a DSP-conformant connector offering the DTR to the Dataspace.
+  - registers Digital Twins (ShellDescriptors in AAS vocabulary) with specificAssetIds including at least one that's known
```

Comment:
AssetAdministrationShellDescriptors if you want to be precise. There is not "ShellDescriptor".

Reply:
with asset IDs (global asset ID and specific asset IDs)